packages/dcos-integration-test/extra/test_dcos_checks.py

```diff
@@ -0,0 +1,25 @@
+import subprocess
+
+
+def invoke_check_binary(target, *args):
+    """
+    Run the dcos-checks binary.
+
+    :param target: dcos-checks subcommand
+    :param args: a list of target arguments passed to a dcos-checks binary
+    :return: combined stdout/stderr, return code
+    """
+    try:
+        output = subprocess.check_output(
+            ["/opt/mesosphere/bin/dcos-checks", "--role", "master", target] + [arg for arg in args],
```

Comment:
@mellenburg Thanks for the context. I'd be OK with shelling out and leaving a TODO.

@darkonie That SGTM too 👍